packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -91,15 +94,20 @@ export const createConfig = async (database: string, client: boolean) => {
             "format": "ts",
             "outputPath": "./src/resolvers"
           }
-        },
-      },
-      dbmigrations: {
-        "connection": JSON.parse(dbConfig),
-        "client": database
+        }
       }
     }
   };
 
+  if(database === "pg") {
+    graphqlConfigJSON.extensions.graphback['dbMigrations'] = {
```

Comment:
Was there bug for postgress? Why this code

Reply:
No I was adding the dbmigrations option when the database option was pg or in fact not mongo db. If it is not pg it will not send dbmigrations option